docs/index.mdx

```diff
@@ -1,26 +1,43 @@
----
-description: ":dart: Dart framework for Core Lightning :dart: "
----
+<div align="center">
+  <h1>lndart.cln</h1>
+
+  <img src="https://github.com/dart-lightning/icons/raw/main/main/res/mipmap-xxxhdpi/ic_launcher.png" />
+
+  <p>
+    <strong> :dart: Dart framework for Core Lightning :dart: </strong>
+  </p>
 
-<img src="https://github.com/dart-lightning/icons/raw/main/main/res/mipmap-xxxhdpi/ic_launcher.png" />
+  <p>
+   <img alt="GitHub Workflow Status" src="https://img.shields.io/github/workflow/status/dart-lightning/clightning.dart/Sanity%20Check?style=flat-square">
```

Comment:
```suggestion
   <img alt="GitHub Workflow Status" src="https://img.shields.io/github/workflow/status/dart-lightning/clightning.dart/Sanity%20Check?style=flat-square"/>
```

This should be fix the docs page

Reply:
This is not magic, I use the https://mdxjs.com/playground/ and it show the error and the line